### PR TITLE
Add news to LPB

### PR DESCRIPTION
**Pull request by andresriveratoro**
https://vajira.max.gov/browse/API-22680

### CODE REVIEW

File: db/migrate/20230213145053_create_news.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class CreateNews < ActiveRecord::Migration[7.0]
+  def change
+    create_table :news do |t|
+      t.string :title
+      t.string :url
+      t.string :category
+
```
Comment (gileswells) [inline db/migrate/20230213145053_create_news.rb:7]: Don't you need a `source` column as well?

File: app/models/news.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class News < ApplicationRecord
+  include Discard::Model
+
+  CATEGORIES = {
+    articles: 'articles',
+    digital_media: 'digital_media',
+    news_releases: 'news_releases'
+  }.freeze
```
Comment (gileswells) [inline app/models/news.rb:10]: The ticket calls out adding news and categories to the database. To me, I would much prefer to see the categories added as a related table with fields that cover the config and on page text shows in the developer portal that is currently coming from [this file](https://github.com/department-of-veterans-affairs/developer-portal/blob/master/src/content/news.yml).

**Review by gileswells — CHANGES_REQUESTED** [review]
Good first pass but I think we can get in some additional improvements while we're kinda holding for webpack / nodejs updates prior to IA work.

**Comment by andresriveratoro** [comment]
Closing in favor of https://github.com/department-of-veterans-affairs/lighthouse-platform-backend/pull/259

**Pull request closed without merging**